api/dispatcher.proto

```diff
@@ -48,6 +49,24 @@ service Dispatcher { // maybe dispatch, al likes this
 	rpc Tasks(TasksRequest) returns (stream TasksMessage) {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-worker" roles: "swarm-manager" };
 	};
+
+	// CreateContainerAttachment allows the node to request the resources
+	// allocation needed for a container attachment on the specific node.
+	rpc CreateContainerAttachment(CreateContainerAttachmentRequest) returns (CreateContainerAttachmentResponse) {
```

Comment:
@mrjana explained it to me this new attachment should not be exposed via the control api, but only to the nodes, given, as I understand, it is going to be for an internal entity node <-> manager (the container attachment which fulfill the requirement gap for running a container to a swarm network), not something which strictly belongs to the regular swarmkit flow for services.